packages/qdrant-loader/tests/unit/core/test_process_graph.py

```
@@ -0,0 +1,210 @@
+    assert edge.properties["project"] == "project-123"
+
+
+@pytest.mark.asyncio
+async def test_process_graph_empty_subgraph_skips_upsert(
+    monkeypatch,
+    pipeline,
+    document,
+):
+    monkeypatch.setattr(
+        "qdrant_loader.core.pipeline.document_pipeline.get_settings",
+        lambda: mock_settings(True),
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)